cluster/cli.sh

```diff
@@ -1,8 +1,9 @@
-source $(dirname "$0")/../hack/common.sh
+#!/bin/bash
+set -e
 
+source hack/common.sh
 test -t 1 && USE_TTY="-it"
-
-source ${KUBEVIRT_DIR}/cluster/$KUBEVIRT_PROVIDER/provider.sh
+source cluster/$KUBEVIRT_PROVIDER/provider.sh
```

Comment:
I believe we have usecases where the KUBEVIRT_DIR environment variable is overwritten. Everything else in this patch looks okay though. Can we revert this line?